lib/features/modeling/behavior/CopyPasteBehavior.js

```diff
@@ -1,71 +0,0 @@
-import inherits from 'inherits';
```

Comment:
Did this go to the `CreateBehavior`?

Reply:
Pasting onto lanes is handled by `CreateBehavior`. Default sequence flows are handled by `BpmnCopyPaste`.